### ISSUE TITLE
Add some more values to the toml configuration file

### ISSUE THREAD

**Issue by etienne-napoleone** [BuildOnViction/tomochain-v1#162]
For the Docker image, those flags are needed but not yet present in the config file:

```
ETHEREUM OPTIONS:
  --keystore                           Directory for the keystore (default = inside the datadir)
  --ethstats value                     Reporting URL of a ethstats service (nodename:secret@host:port)
  --identity value                     Custom node name

API AND CONSOLE OPTIONS:
  --rpc                  Enable the HTTP-RPC server
  --ws                   Enable the WS-RPC server
  --wsaddr value         WS-RPC server listening interface (default: "localhost")
  --wsport value         WS-RPC server listening port (default: 8546)
  --wsapi value          API's offered over the WS-RPC interface

NETWORKING OPTIONS:
  --port value          Network listening port (default: 30303)
  --nat value           NAT port mapping mechanism (any|none|upnp|pmp|extip:<IP>) (default: "any")

LOGGING AND DEBUGGING OPTIONS:
  --verbosity value         Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail (default: 3)
```

**Comment by ngtuna** [comment]
👍 will be added soon

**Comment by ngtuna** [comment]
tested. I am going to close this

**Comment by ngtuna** [comment]
fixed via #168
